### PR TITLE
zh-CN: gpu.md

### PR DESCRIPTION
**Pull request by Bryan-Kan**
Translate gpu.md

### CODE REVIEW

File: site/zh-cn/lite/performance/gpu.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+点击 Run 按钮来运行应用程序。当你运行应用程序的时候你会看到一个启用 GPU 的按钮。将应用程序从量化模式改为浮点模式后点击 GPU 按钮后，程序将在 GPU 上运行。
+
+![运行 Android gpu 演示应用程序和切换到 GPU](images/android_gpu_demo.gif)
+
+### iOS (使用 XCode)
+
+如果需要一个分步教程, 请观看
+[适用于 iOS 的实验性 GPU 代理](https://youtu.be/Xkhgre8r5G0) 的视频。
+
+注意：这需要XCode 10.1或者更高版本
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
注意：这需要 XCode 10.1 或者更高版本
```

File: site/zh-cn/lite/performance/gpu.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+#### 第一步. 获取演示应用程序的源码并确保它已被编译
+
+遵照我们的 iOS 演示应用程序[教程](https://www.tensorflow.org/lite/demo_ios)。这会告诉你没有修改的iOS相机应用程序是如何在我们的手机上运行的。
+
+#### 第二部. 修改 Podfile 文件来使用 TensorFlow Lite GPU CocoaPod
+
+我们构建了一个包含GPU代理的二进制CocoaPod文件。如果需要切换到工程并使用它，修改
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
我们构建了一个包含 GPU 代理的二进制 CocoaPod 文件。如果需要切换到工程并使用它，修改
```

File: site/zh-cn/lite/performance/gpu.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+`tensorflow/tensorflow/lite/examples/ios/camera/Podfile` 文件来使用  `TensorFlowLiteGpuExperimental` 的 pod 替代 `TensorFlowLite`.
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
`tensorflow/tensorflow/lite/examples/ios/camera/Podfile` 文件来使用  `TensorFlowLiteGpuExperimental` 的 pod 替代 `TensorFlowLite`。
```

File: site/zh-cn/lite/performance/gpu.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+```
+target 'YourProjectName'
+  # pod 'TensorFlowLite', '1.12.0'
+  pod 'TensorFlowLiteGpuExperimental'
+```
+
+#### 第三步. 启用GPU代理
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
#### 第三步. 启用 GPU 代理
```

File: site/zh-cn/lite/performance/gpu.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+为了确保代码会使用GPU代理，你需要将 `CameraExampleViewController.h` 的
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
为了确保代码会使用 GPU 代理，你需要将 `CameraExampleViewController.h` 的
```

File: site/zh-cn/lite/performance/gpu.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+`TFLITE_USE_GPU_DELEGATE` 从0修改为1 .
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
`TFLITE_USE_GPU_DELEGATE` 从 0 修改为 1 。
```

File: site/zh-cn/lite/performance/gpu.md [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+#define TFLITE_USE_GPU_DELEGATE 1
+```
+
+#### 第四步. 编译和运行演示应用程序
+
+如果你完成了上面的步骤，你应该已经可以运行这个应用程序了。
+
+#### 第五步. 发布模式
+
+你在第四步是在调试模式下运行的应用程序，为了获得更好的性能表现，你应该使用适当的最佳  Metal 设置将应用程序改为发布版本。特别需要注意的是，需要修改这些设置 `Product > Scheme > Edit
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
你在第四步是在调试模式下运行的应用程序，为了获得更好的性能表现，你应该使用适当的最佳 Metal 设置将应用程序改为发布版本。特别需要注意的是，需要修改这些设置 `Product > Scheme > Edit
```

File: site/zh-cn/lite/performance/gpu.md [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+Scheme...`，选择 ` Run `，在 ` Info ` 一栏，修改 ` Build Configuration `，从 `Debug ` 改为 ` Release `，取消选择 ` Debug executable`。
+
+![设置发布](images/iosdebug.png)
+
+然后点击 `Options` 栏然后将 `GPU Frame Capture` 修改成 `Disabled`，并将 `Metal API Validation` 修改成 `Disabled`。
+
+![设置metal选项](images/iosmetal.png)
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
![设置 metal 选项](images/iosmetal.png)
```

File: site/zh-cn/lite/performance/gpu.md [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+
+最后需要确保发布版本只能在64位系统上构建。在 `Project
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
最后需要确保发布版本只能在 64 位系统上构建。在 `Project
```

File: site/zh-cn/lite/performance/gpu.md [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+  if (interpreter->Invoke() != kTfLiteOk) return false;
+  ReadFromOutputTensor(interpreter->typed_output_tensor<float>(0));
+}
+
+// 清理
+interpreter = nullptr;
+DeleteGpuDelegate(delegate);
+```
+
+## 支持的模型和Ops
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
## 支持的模型和 Ops
```

File: site/zh-cn/lite/performance/gpu.md [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+
+在GPU代理发布后，我们提供了少数可以在后端运行的模型：
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
在 GPU 代理发布后，我们提供了少数可以在后端运行的模型：
```

File: site/zh-cn/lite/performance/gpu.md [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+* [DeepLab 分割 (257x257)](https://ai.googleblog.com/2018/03/semantic-image-segmentation-with.html) [[下载]](https://storage.googleapis.com/download.tensorflow.org/models/tflite/gpu/deeplabv3_257_mv_gpu.tflite)
+<br /><i>(将输入图像的每个像素指定语义标签（例如，狗，猫。汽车的图像分割模型)</i>
+* [MobileNet SSD 物体检测](https://ai.googleblog.com/2018/07/accelerated-training-and-inference-with.html) [[下载]](https://storage.googleapis.com/download.tensorflow.org/models/tflite/gpu/mobile_ssd_v2_float_coco.tflite)
+<br /><i>(用于检测多个带有边框的对象的图像分类模型)</i>
+* [PoseNet用于姿势估计](https://github.com/tensorflow/tfjs-models/tree/master/posenet) [[下载]](https://storage.googleapis.com/download.tensorflow.org/models/tflite/gpu/multi_person_mobilenet_v1_075_float.tflite)
+<br /><i>(用于估计图像或视频中人物的姿势的视觉模型)</i>
+
+如果需要完整的支持的 Ops 的列表，请看[进阶文档](gpu_advanced.md)。
+
+## 不支持的模型和ops
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
## 不支持的模型和 ops
```

File: site/zh-cn/lite/performance/gpu.md [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+
+如果一些 ops 并不支持 GPU 代理，框架只会在 GPU 上运行图形的一部分，剩下的部分会在 CPU 上运行。因为这会导致 CPU/GPU 同时出现很高的使用率，像这样的分开执行模式会导致运行起来比整个网络在CPU上运行要慢。在这种情况下，用户会收到一个像这样的警告：
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]:
```suggestion
如果一些 ops 并不支持 GPU 代理，框架只会在 GPU 上运行图形的一部分，剩下的部分会在 CPU 上运行。因为这会导致 CPU/GPU 同时出现很高的使用率，像这样的分开执行模式会导致运行起来比整个网络在 CPU 上运行要慢。在这种情况下，用户会收到一个像这样的警告：
```

File: site/zh-cn/lite/performance/gpu.md [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+
+```
+WARNING: op code #42 cannot be handled by this delegate.
```
Comment (Mr-Linus) [inline site/zh-cn/lite/performance/gpu.md]: 这个不需要翻译吗

**Comment by Mr-Linus** [comment]
LGTM
cc @lamberta

**Comment by Mr-Linus** [comment]
@Bryan-Kan Update CLAs.Please comment the PR and let the @googlebot check CLAs again.

File: site/zh-cn/lite/performance/gpu.md (hunk 14 above)
Comment (Bryan-Kan) [reply site/zh-cn/lite/performance/gpu.md]: 已修改

**Comment by lamberta** [comment]
Confirmed CLA for username. Used a different email for signing the CLA than the email used for the commit. May need to update your `.gitconfig`

**Pull request merged**